browser/src/layer/tile/ImpressTileLayer.js

```diff
@@ -77,6 +77,11 @@ L.ImpressTileLayer = L.CanvasTileLayer.extend({
 			TileManager.refreshTilesInBackground();
 			TileManager.update();
 		}
+
+		if (['DrawPage', 'NotesPage', 'MasterPage'].includes(e.detail.context)) {
+			app.socket.sendMessage('status');
```

Comment:
we could also check if context was really changed
https://github.com/CollaboraOnline/online/blob/ae8300e5f556ce914e2c9256b6b2276a53520678/browser/src/control/Control.Notebookbar.js#L572

see comment in the code above `When user modifies the content of the notes, the context is changed again`
this is I'm talking about - it can render previews too often possibly?